Add a Github pull request template

toniecrumley opened the pull request:
## Summary
<!-- Describe your science -->
Add a CAPTURE specific pull request template.  See Github [Documentation](https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/creating-a-pull-request-template-for-your-repository#adding-a-pull-request-template)

toniecrumley committed:
Add a Github pull request template

tsoelter commented on .github/pull_request_template.md:
```diff
@@ -0,0 +1,69 @@
+- [ ] Setup instructions have been provided.
+- [ ] Test instructions have been provided.
+- [ ] Cleanup instructions have been provided.
+- [ ] Primary and secondary reviewers have been requested to review this
```
Not sure if you want to remove the "primary and secondary" portion, since that's not really applicable here? But it also feels so minor, maybe only change if others request changes on this.

toniecrumley replied:
This has been changed to "At least two reviewers..."

toniecrumley committed:
Fix code review request check list item wording

toniecrumley committed:
Add a blank line to the end of code blocks

This makes the code blocks execute the last line when the copy icon
is used in Github to copy and paste the code into a terminal prompt.

lizzyjoan commented on .github/pull_request_template.md:
```diff
@@ -0,0 +1,69 @@
+- [ ] No merge conflicts exist. Merge conflicts are indicated by the "This
+  branch has conflicts that must be resolved" message in the GitHub pull
+  request status box above the "Merge pull request" button.
+- [ ] The CHANGELOG has been updated with the planned version number.
```
Would you want to add a checklist item like "repo README updated if applicable" for new functionalities? Just a thought

toniecrumley replied:
Yes, great catch.  I added that.

toniecrumley committed:
Add check list items for updating documentation
